### PR TITLE
DrivAerML: Fourier+no-EMA — 4L/256d model capacity increase

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
On AirfRANS, increasing model capacity from 3L/192d to 4L/256d (with 4 heads) improved results from 0.2597 to 0.2387 — a 8.1% improvement (PR #2478). DrivAerML is a 3D problem with significantly higher geometric complexity than 2D AirfRANS. Increasing model capacity may yield similar or larger gains on DrivAerML. The default architecture is 3L/192d; we test 4L/256d/4H.

## Instructions

Run two parallel jobs to test capacity increase and compare:

**Run 1 — 4L/256d/4H (increased capacity):**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=0 python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 5e-4 \
  --cosine_t_max 30 \
  --no-use-ema \
  --enable-fourier \
  --model-layers 4 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group ray-drivaerml-capacity &
```

**Run 2 — 5L/320d/4H (even more capacity):**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=1 python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 5e-4 \
  --cosine_t_max 30 \
  --no-use-ema \
  --enable-fourier \
  --model-layers 5 \
  --model-hidden-dim 320 \
  --model-heads 4 \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group ray-drivaerml-capacity
```

Wait for both to complete. Report `val_primary/surface_rel_l2_pct` for each run, epochs completed, peak VRAM usage, and W&B run IDs. Note if larger model OOMs or significantly reduces epochs per hour.

**Critical:** You must pass `--epochs 999` — the default is only 2 epochs due to a hardcoded bug in the training script.

## Baseline
- **Primary metric:** `val_primary/surface_rel_l2_pct` (lower is better)
- **Current best (val):** 51.35% with default 3L/192d (PR #2475, W&B run: 5ncrjm32)
- **Current best (test):** 52.06%
- **AirfRANS precedent:** 4L/256d improved AirfRANS from 0.2597 → 0.2387 (-8.1%)
- **Reproduce baseline:** `cd target/icml2026 && python train.py --dataset drivaerml --optimizer adamw --lr 5e-4 --cosine_t_max 30 --no-use-ema --enable-fourier --epochs 999 --batch-size 1 --drivaerml-train-surface-points 50000 --drivaerml-eval-surface-points 50000 --max-train-batches 394 --max-eval-batches 200`

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2527 because Duplicate assignment — closing extra PR created by parallel agent. Student already has a primary experiment assigned..

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2527 because Closing duplicate — student ray already has active PR #2518 assigned..
